stac_fastapi/api/stac_fastapi/api/app.py

```diff
@@ -75,7 +146,7 @@ class StacApi:
     search_request_model: Type[Search] = attr.ib(default=STACSearch)
     search_get_request: Type[SearchGetRequest] = attr.ib(default=SearchGetRequest)
     item_collection_uri: Type[ItemCollectionUri] = attr.ib(default=ItemCollectionUri)
-    response_class: Type[Response] = attr.ib(default=JSONResponse)
+    response_class: Type[Response] = attr.ib(default=ORJSONResponse)
```

Comment:
This changes the default JSONResponse value to ORJSONResponse, which seems like an orthogonal change to this PR. And, orjson isn't an explicit dependency to anything but pgstac, so either orjson should be added as a dependency to the `api` project or it shouldn't be used in this code file (applies also to the base class of `VndOaiResponse), as it would break installations that only relied on the sqlalchemy backend.